From a17a92d47fbd3eb33ae30806d8b12a83ff01acbb Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Wed, 16 Jun 2021 20:40:00 -0700
Subject: [PATCH 120/196] Update genesis blocktime and nonce for mainnet (#196)

Update Genesis block to have a time of the summer solstice in 2021, and
a block hash starting with `abc`.
---
 src/chainparams.cpp | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index 2b81e1667..0725e82db 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -128,12 +128,13 @@ public:
         m_assumed_chain_state_size =
             ChainParamsConstants::MAINNET_ASSUMED_CHAINSTATE_SIZE;
 
-        genesis = CreateGenesisBlock(0x1c100000, 1619549570, 173024572809ull);
+        genesis =
+            CreateGenesisBlock(0x1c100000, 1624246260, 7146261898250975403ull);
         consensus.hashGenesisBlock = genesis.GetHash();
         assert(genesis.GetSize() == 379);
         assert(consensus.hashGenesisBlock ==
-               uint256S("0000000004d77f4cf08a2c37867778f1a3d6272f1e5e3c59ee20ed"
-                        "b739a52c5d"));
+               uint256S("000000000abc0cde58ee7e919d3d4de183e6844add1fd5d14b4eac"
+                        "89d958f470"));
         assert(genesis.hashMerkleRoot ==
                uint256S("37f392d88f70cdada6d366a25a7ef90b6711bf2d6b5ffea4f39727"
                         "dcb90af34c"));
-- 
2.39.5 (Apple Git-154)

